src/main/java/teammates/common/util/Const.java

```diff
@@ -800,6 +800,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String EMAIL_COURSE = "course";
         public static final String EMAIL_FEEDBACK = "feedback";
         public static final String EMAIL_TYPE = "type";
+        public static final String EMAIL_SEND_CHECK = "isSendEmail";
```

Comment:
Similar to the other PR, drop the `is-` prefix and use all lowercase, i.e `sendemail`. Also, what `sendemail` is an ambiguous parameter; what "email" are we talking about here?